.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```
@@ -0,0 +1,103 @@
+name: Bug report
+description: Create a report to help us improve
+#labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **THIS IS NOT A SUPPORT CHANNEL!**
+        **IF YOU HAVE SUPPORT QUESTIONS ABOUT RUNNING OR CONFIGURING YOUR OWN HOME SERVER**,
+        please ask in **[#synapse:matrix.org](https://matrix.to/#/#synapse:matrix.org)** (using a matrix.org account if necessary).
+
+        If you want to report a security issue, please see https://matrix.org/security-disclosure-policy/
+
+        This is a bug report template. By following the instructions below and
+        completing the sections with your information, you will help the us to get all
+        the necessary data to fix your issue.
+
+        You can also preview your report before submitting it. You may remove sections
+        that aren't relevant to your particular case.
```

```suggestion
        You can also preview your report before submitting it.
```

Since you can't remove sections in forms. We could `s/remove/ignore/`, but I think this would just lead to issues with empty sections where it's not clear whether the reporter deliberately ignored it because it's not relevant or they missed it.